src/pages/resultsView/oncoprint/TracksMenu.tsx

```diff
@@ -60,9 +67,27 @@ export const COUNT_PADDING_WIDTH = 17;
 export default class TracksMenu extends React.Component<IAddTrackProps, {}> {
     @observable tabId: Tab | string = Tab.CLINICAL;
 
+    private genericAssayEntityOptionsReaction: IReactionDisposer;
+
+    private selectedGenericAssayProfileIdByType = observable.map<
+        string,
+        string
+    >({}, { deep: true });
+
     constructor(props: IAddTrackProps) {
         super(props);
         makeObservable(this);
+        // Using the first profile as default profile option to initialize selectedGenericAssayProfileIdByType
+        this.genericAssayEntityOptionsReaction = when(
```

Comment:
not sure why this reaction is necessary

Reply:
@alisman Thanks for this comment. I've simplified the code deleting the reaction